xtable-core/src/main/java/org/apache/xtable/delta/DeltaValueConverter.java

```diff
@@ -242,18 +244,11 @@ private static Object castObjectToInternalType(Object value, InternalType valueT
     return value;
   }
 
-  private static BigDecimal numberTypeToBigDecimal(Object value) {
+  private static BigDecimal numberTypeToBigDecimal(Object value, InternalSchema schema) {
     // BigDecimal is parsed as Integer, Long, BigInteger and double if none of the above.
```

Comment:
Do we still need this comment ?     
`// BigDecimal is parsed as Integer, Long, BigInteger and double if none of the above.`